### DISCUSSION TITLE
How to customize the item label on Relationship field's options?

### DISCUSSION THREAD

**Discussion by ChrisGV04** [payloadcms/payload#3216] [Q&A]
Hi! I'm developing a website for a customer and they have a "Careers" page that shows the job listings that they are currently hiring. Since they might reuse the job listings at different times, I created a collection called `Jobs` where we can add the information about each job (name, location, salary, etc.). Then we have the `CareersPage` global document which contains the information shown on that page and a Relationship field to choose which job listings to show.

When we click on the Relationship field to choose an option, I get the list of jobs just as expected. However, we have certain job listings with the same name but for different locations, so we cannot identify which job is which easily from the select menu because they have the same name and the Jobs collection has `admin.useAsTitle: "name"` . See picture for reference:

<img width="765" alt="Screenshot 2023-08-23 at 12 31 02 p m" src="https://github.com/payloadcms/payload/assets/47041342/c21a1275-5f44-4583-82e0-86a7cbc35a42">

In the example shown above, 2 job listings have the same name but each one is for a different clinic.

Is there a way to **only customize the label inside that field** without having to create a full custom component to render that select field? 

I would like to format the label to be `name - location` to identify them easily if possible. Thank you in advance!

**Comment by r1tsuu** [comment]
Hello, there's no something like that now, i've added a PR #3232
